metrics/dbmetrics.go

```diff
@@ -102,3 +116,50 @@ func (e *dbCallMetrics) elapsed() time.Duration {
 	}
 	return e.endTime.Sub(e.startTime)
 }
+
+// NewPostgreDBMetrics returns new PostgreDBMetrics.
+func NewPostgreDBMetrics(metricsProvider Provider, dbName string) *PostgreDBMetrics {
+	l := []string{dbMetricLabelInstance}
+	MaxOpenConnections := metricsProvider.NewHistogram("postgres_db_stat_max_open_connections",
```

Comment:
I don't see these variables are used anywhere outside this functions. I think it's better to rename it to unexported variables. What do you think?

Reply:
it will be called by the app that use this lib. wdyt
for example:
`server.dbPostgreMetric = metrics.NewPostgreDBMetrics(metrics.DefaultProvider, dbName)`

Reply:
I mean the `MaxOpenConnections` variable. Is it used outside the `NewPostgreDBMetrics` func?

Reply:
@thoriqsatriya i see, code updated. wdyt?